content/02.delete-me.md

```diff
@@ -1,15 +1,165 @@
+two spaces at end of line.
+
+Putting each sentence on its own line has numerous benefits with regard to [editing](https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line) and [version control](https://rhodesmill.org/brandon/2012/one-sentence-per-line/).
+
+1. Ordered list item
+2. Ordered list item
+    a. Sub-item
+    b. Sub-item
+        i. Sub-sub-item
+3. Ordered list item
+    a. Sub-item
+
+- List item
+- List item
+- List item
+
+superscript<sup>2</sup>
+
+subscript<sub>2n+1</sub>
+
+unicode superscript™
+
+unicode subscriptₓ
+
+**Links and Citations**
+
+Bare URL link: <https://manubot.org>
+
+[Link with text](https://www.manubot.org)
+
+[Link with hover text](https://www.manubot.org "Manubot Homepage")
+
+[Link styled as a button](https://www.manubot.org "Manubot Homepage"){.button}
+
+[Link by reference][manubot homepage]
+
+[Manubot Homepage]: https://www.manubot.org
+
+Citation via DOI: [@doi:10.15363/thinklab.4]
+
+Citation by tag: [@tag:deep-review]
+
+Manual URL citation: [@url:https://greenelab.github.io/meta-review]
+
+**Figures and Tables**
+
+Figure @fig:square-image
+
+Figure @fig:wide-image
+
+Figure @fig:tall-image
+
+Figure @fig:vector-image
+
+Table @tbl:bowling-scores
+
+**Quotes and Code**
+
+> Quoted text
+
+> Quoted block of text  
+>
+> Two roads diverged in a wood, and I—  
+> I took the one less traveled by,  
+> And that has made all the difference.  
+
+Code `in the middle` of normal text, aka `inline code`
+
+```c++
+// code block with C++ syntax highlighting
+int main() {
+    // a string too long to fit within page
+    cout << "It was the best of times, it was the worst of times, it was the age of wisdom, it was the age of foolishness";
+    return 0;
+}
+```
+
+```
+// code block with no syntax highlighting
+int count = 0;
+function add() {
+	count++;
+}
+```
+
+![A square image at actual size and with a bottom caption](https://raw.githubusercontent.com/greenelab/manubot-resources/master/images/square.png "Square image"){#fig:square-image}
+
+![An image too wide to fit within page at full size](https://raw.githubusercontent.com/greenelab/manubot-resources/master/images/wide.png "Wide image"){#fig:wide-image}
+
+![A tall image with a specified height](https://raw.githubusercontent.com/greenelab/manubot-resources/master/images/tall.png "Tall image"){#fig:tall-image height=3in}
+
+![A vector `.svg` image with `sanitize=true` to load from Github](https://raw.githubusercontent.com/manubot/resources/master/images/vector.svg?sanitize=true "Vector image"){#fig:vector-image height=2in}
+
+| *Bowling Scores* | Jane          | John          | Alice         | Bob           |
+|:-----------------|:-------------:|:-------------:|:-------------:|:-------------:|
+| Game 1 | 150 | 187 | 210 | 105 |
+| Game 2 |  98 | 202 | 197 | 102 |
+| Game 3 | 123 | 180 | 238 | 134 |
+
+Table: A table with a top caption and specified relative column widths {#tbl:bowling-scores}
```

Comment:
Can add newline before `{#tbl:bowling-scores}` for one sentence per line.

See how done in [Sci-Hub manuscript](https://raw.githubusercontent.com/greenelab/scihub-manuscript/8e538a98e4f16bab632c00efb1fb4f1b23a1c122/content/02.manuscript.md):

```markdown
Table: **Coverage for the ten journals with the most articles.**
The total number of articles published by each journal is noted in the Crossref column.
The table provides the number (Sci-Hub column) and percentage (Coverage column) of these articles that are in Sci-Hub's repository.
{#tbl:top-ten-journals}
```